availability.php

```diff
@@ -76,6 +76,6 @@
     <form action="availability.php" method="post">
       ' . daily_matrix ( $date, $users ) . '
     </form>
-    ' . print_trailer ( false, true, true );
+    ' . print_trailer ( 0, 1, 1 );
```

Comment:
You can try out PHP 7.1 with the a docker container: https://github.com/craigk5n/webcalendar-docker

That's how I've been testing PHP 7. I generally just use "php -l" on all the php files. But I'd love to use Selenium IDE to create some automated tests... just haven't got around to it.